math/wxMaxima/Portfile

```diff
@@ -1,59 +1,110 @@
-PortSystem      1.0
-PortGroup       active_variants 1.1
-PortGroup       wxWidgets 1.0
-
-name            wxMaxima
-version         15.04.0
-revision        0
-maintainers     nomaintainer
-license         gpl
-platforms       darwin
-categories      math aqua
-description     Graphical user interface for Maxima
-long_description \
-                wxMaxima is a cross platform GUI for the computer algebra \
-                system maxima based on wxWidgets.
-homepage        http://wxmaxima.sourceforge.net/
-master_sites    sourceforge:project/[string tolower ${name}]/${name}/${version}
-livecheck.regex (?:[string tolower ${name}]|${name})-(\[a-z0-9.\]+)${extract.suffix}
-checksums       rmd160  04295f10ccd074d7069d00b1a0fe41884817687c \
-                sha256  0005029798703470333309fda29c731e79c85de1c7fc09b16cb87045a03aa4ba
-
-# Upstream changed the distname's spelling some time after 13.x.
-distname        wxmaxima-${version}
-
-wxWidgets.use   wxWidgets-3.0
-
-patchfiles      patch-src_main.cpp.diff \
-                patch-src_Dirstructure.h.diff \
-                patch-src_wxMaxima.cpp.diff
-
-depends_lib     port:libiconv \
-                port:libsdl \
-                port:libxml2 \
-                port:${wxWidgets.port} \
-                port:zlib
-depends_run     port:maxima
-
-require_active_variants gnuplot wxwidgets
-
-post-patch {
-    reinplace "s|@@PREFIX@@|${prefix}|" ${worksrcpath}/src/main.cpp
+PortSystem          1.0
+
+PortGroup           active_variants 1.1
+PortGroup           wxWidgets 1.0
+PortGroup           github 1.0
+
+name                wxMaxima
+version             18.02.0
+maintainers         nomaintainer
+license             GPL-2
+# openssl is a dependency of curl in cmake
+license_noconflict  openssl
+platforms           darwin
+categories          math aqua
+description         Graphical user interface for Maxima based on wxWidgets
+homepage            http://andrejv.github.io/wxmaxima/
+
+subport wxMaxima-devel {
+    conflicts       wxMaxima
+    long_description ${description}: \
+                    Provides the development version, which is typically updated every day.
+
+    # git describe --tags : Version-16.12.x-1834-gb32fed0
+    version         18.02-dev-20180616
+    github.setup    andrejv wxmaxima b32fed0d535c05d589d1f573d2c0f4f839180d18
```

Comment:
You are getting closer. :) The version line needs to go after the github.setup line because it is getting overridden.

Reply:
I see. I have misunderstood. I will move the version line.